src/test/ui/suggestions/lifetimes/missing-lifetimes-in-signature.stderr

```diff
@@ -1,118 +1,11 @@
 error[E0261]: use of undeclared lifetime name `'a`
-  --> $DIR/missing-lifetimes-in-signature.rs:37:11
+  --> $DIR/missing-lifetimes-in-signature.rs:36:11
    |
 LL | fn baz<G: 'a, T>(g: G, dest: &mut T) -> impl FnOnce() + '_
    |        -  ^^ undeclared lifetime
    |        |
    |        help: consider introducing lifetime `'a` here: `'a,`
 
-error[E0759]: `dest` has an anonymous lifetime `'_` but it needs to satisfy a `'static` lifetime requirement
```

Comment:
Just lots of errors gone here. I assume same problem as other two tests: these errors probably get emitted in the `wf_checking` bit, which we don't reach because of the first error.